warehouse/query-core/src/main/java/datawave/query/iterator/QueryIterator.java

```diff
@@ -1245,10 +1244,10 @@ protected DocumentProjection getProjection() {
         DocumentProjection projection = new DocumentProjection(this.isIncludeGroupingContext(), this.isReducedResponse(), isTrackSizes());
         
         if (this.useWhiteListedFields) {
```

Comment:
Do we want to update this language as well?

Reply:
sorry.  should have added these comments on the first pass.  approval still stands.

Reply:
No problem. I'm limiting the scope to just the Projection classes, updating the QueryIterator, various querylogics, and configs will be another unit of work (with a higher cost to integrate)